src/observers/InformationMessageObserver.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as ObservableEvent from "../omnisharp/loggingEvents";
+import { vscode } from '../vscodeAdapter';
+
+export class InformationMessageObserver {
```

Comment:
Why does the info message observer not need to debounce when the warning one does? 
Also, why is there not an error message observer (maybe it was added in another PR)?

Generally, it seems like we might be painting ourselves into a corner here. If for some reason, we want to be able to configure the errorlevel of a message, we now have to go teach each error level observer how to print that message. Maybe it would be clearer to only have one WindowPostMessageObserverThingy and have it know how to translate event -> VS Code Warning level?

Reply:
The error event and MSbuildDiagnostics event shows the warning message hence they are both present in the warning message observer